### DISCUSSION TITLE
ITMS-90901: Missing full-screen support for the latest iPad mini display. Apple Account Error

### DISCUSSION THREAD

**Discussion by anujdivesh** [expo/expo#15935] [Archived: Questions]
Hi, I just submitted my app to my appstoreconnect account, it returned with the following issue:
```
TMS-90901: Missing full-screen support for the latest iPad mini display. The “ExpoKitApp.app” bundle includes UIRequiresFullScreen=YES in the Info.plist or supports only a subset of UISupportedInterfaceOrientations, and is built with the 14.1 SDK. To take advantage of the full screen size, recompile with Xcode 13 and the SDK for iPadOS 15 or later.
```
Thanks in advance

**Comment by ankitvat** [comment]
Facing the same Issue , tried adding UIRequiresFullSreen: true in infoPlist ... didnt work ..

**Reply by anujdivesh** [reply]
Hi @ankitvat I seem to got it working, but not sure if this is the fix, I updated my app to latest sdk and included in the app.json
```
"ios": {
      "supportsTablet": true,
      "infoPlist": {
        "UIRequiresFullScreen": true
      },
      "bu.....
```
can you see if it works?

**Reply by sdinesmvc** [reply]
@anujdivesh  Which expo version are you upgraded. how you upgraded please share me the steps.

**Reply by ankitvat** [reply]
@anujdivesh .. Actually this fix still gives the error since Expo doesn't allow you to open Xcode. However this error is just a warning and won't affect the build or deployment on App Store.

As for @sdinesmvc .. Expo version is 41.0 I guess... Would recommend to use RN CLI instead of Expo

**Reply by sdinesmvc** [reply]
@ankitvat  if I switching to RN CLI, is a huge work right?. whether Expo version is 41.0 version with the below configuration it will work right? 
"ios": {
      "supportsTablet": true,
      "infoPlist": {
        "UIRequiresFullScreen": true
      },
      "bu.....

**Comment by anujdivesh** [comment]
Hi @sdinesmvc, When I was using SDK 42 i got that warning, once i upgraded to SDK 44 the warning seemed to have disappeared. @ankitvat  yes, it wont affect deployment.
